curator/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:latest
+
+
+RUN apk --update add python py-pip && \
+    pip install elasticsearch-curator && \
+     rm -rf /var/cache/apk/*
+
+ADD entrypoint.sh /entrypoint.sh
```

Comment:
ADD should be reserved for copying files from HTTP URLs. Besides you could simply copy to `/` since you don't rename the file.